<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QWidget" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>第五人格bp系统-后台</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>521</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>队伍信息</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华康POP</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>监管者队伍</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>lEdtHun</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华康POP</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>求生者队伍</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>lEdtSur</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="pButSwapTeam">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>30</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>交换位置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEdtSur">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lEdtHun">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>271</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>选图</string>
   </property>
   <widget class="QComboBox" name="cBoxMap">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>34</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>511</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>ban求生者</string>
   </property>
   <widget class="QComboBox" name="cBoxSurBan1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ban1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>cBoxSurBan1</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="labSurBan1Pic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ban2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>cBoxSurBan2</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cBoxSurBan2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labSurBan2Pic">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ban3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>cBoxSurBan3</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cBoxSurBan3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labSurBan3Pic">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ban4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>cBoxSurBan4</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cBoxSurBan4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labSurBan4Pic">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ban5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>cBoxSurBan5</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cBoxSurBan5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labSurBan5Pic">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cBoxSurBan6">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ban6</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>cBoxSurBan6</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="labSurBan6Pic">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>130</y>
     <width>391</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>ban监管者</string>
   </property>
   <widget class="QComboBox" name="cBoxHunBan1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ban1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>cBoxHunBan1</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="labHunBan1Pic">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ban2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>cBoxHunBan2</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cBoxHunBan2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labHunBan2Pic">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ban3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>cBoxHunBan3</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cBoxHunBan3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labHunBan3Pic">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ban4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>cBoxHunBan4</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cBoxHunBan4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labHunBan4Pic">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pBtnGen">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>40</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>生成bp界面</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>391</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>pick求生者</string>
   </property>
   <widget class="QComboBox" name="cBoxSurPick1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pick1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>cBoxSurPick1</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="labSurPick1Pic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pick2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>cBoxSurPick2</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cBoxSurPick2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labSurPick2Pic">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pick3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>cBoxSurPick3</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cBoxSurPick3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labSurPick3Pic">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pick4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>cBoxSurPick4</cstring>
    </property>
   </widget>
   <widget class="QComboBox" name="cBoxSurPick4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labSurPick4Pic">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>111</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>pick监管者</string>
   </property>
   <widget class="QComboBox" name="cBoxHunPick">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pick</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buddy">
     <cstring>cBoxHunPick</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="labHunPickPic">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pBtnRes">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>80</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lEdtSur</tabstop>
  <tabstop>pButSwapTeam</tabstop>
  <tabstop>lEdtHun</tabstop>
  <tabstop>cBoxMap</tabstop>
  <tabstop>pBtnGen</tabstop>
  <tabstop>pBtnRes</tabstop>
  <tabstop>cBoxSurBan1</tabstop>
  <tabstop>cBoxSurBan2</tabstop>
  <tabstop>cBoxSurBan3</tabstop>
  <tabstop>cBoxSurBan4</tabstop>
  <tabstop>cBoxSurBan5</tabstop>
  <tabstop>cBoxSurBan6</tabstop>
  <tabstop>cBoxHunBan1</tabstop>
  <tabstop>cBoxHunBan2</tabstop>
  <tabstop>cBoxHunBan3</tabstop>
  <tabstop>cBoxHunBan4</tabstop>
  <tabstop>cBoxSurPick1</tabstop>
  <tabstop>cBoxSurPick2</tabstop>
  <tabstop>cBoxSurPick3</tabstop>
  <tabstop>cBoxSurPick4</tabstop>
  <tabstop>cBoxHunPick</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
